caikit/runtime/__init__.py

```diff
@@ -11,3 +11,6 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+# Local
+from .service_factory import get_request, get_train_params, get_train_request
```

Comment:
Small note: I know that in `caikit.runtime` we've NOT done proactive imports. I think this is a fine thing to do since it surfaces these functions at a module level, but we do want to be a little careful about what (if any) import-time code gets run by performing these proactive imports.